README.md

```diff
@@ -1,118 +1,24 @@
-<p align="center">
-  <a href="https://celo.org/">
-    <img src="https://i.imgur.com/fyrJi0R.png" alt="celo logo" title="Go to celo.org" width="600" style="border:none;"/>
-  </a>
-</p>
+# Valora Mobile App
 
-**Official repository for Valora**
+Valora is a mobile wallet focused on making global peer-to-peer
+payments simple and accessible to anyone. It supports the Celo
+Identity Protocol which allows users to verify their phone number and
+send payments to their contacts.
 
-Valora is a mobile wallet focused on making global peer-to-peer payments simple and accessible to anyone. It supports the Celo Identity Protocol which allows users to verify their phone number and send payments to their contacts.
+## Build Valora Locally
 
-- [valoraapp.com](https://valoraapp.com)
-- Platforms: iOS, Android
-- Maintainers: [cLabs](https://clabs.co)
+To setup Valora locally, follow [setup instructions](https://github.com/valora-inc/wallet/tree/main/packages/mobile).
 
-<!-- row 1 - status -->
-
-![example workflow](https://github.com/celo-org/wallet/actions/workflows/check.yml/badge.svg)
-![example workflow](https://github.com/celo-org/wallet/actions/workflows/e2e.yml/badge.svg)
-![example workflow](https://github.com/celo-org/wallet/actions/workflows/test.yml/badge.svg)
-[![Codecov](https://img.shields.io/codecov/c/github/celo-org/celo-monorepo)](https://codecov.io/gh/celo-org/celo-monorepo)
-[![GitHub contributors](https://img.shields.io/github/contributors/celo-org/celo-monorepo)](https://github.com/celo-org/celo-monorepo/graphs/contributors)
-[![GitHub commit activity](https://img.shields.io/github/commit-activity/w/celo-org/celo-monorepo)](https://github.com/celo-org/celo-monorepo/graphs/contributors)
-[![GitHub Stars](https://img.shields.io/github/stars/celo-org/celo-monorepo.svg)](https://github.com/celo-org/celo-monorepo/stargazers)
-![GitHub repo size](https://img.shields.io/github/repo-size/celo-org/celo-monorepo)
-[![GitHub](https://img.shields.io/github/license/celo-org/celo-monorepo?color=blue)](https://github.com/celo-org/celo-monorepo/blob/master/LICENSE)
```

Comment:
Would love to keep our badges, but use the correct repo instead.

Reply:
I can re-add them.  Some of them are superfluous (e.g., github stars are listed elsewhere on the page rendering `README.md`), and some of them aren't actionable (e.g., what's a potential contributor supposed to do if a workflow is failing?).  We can discuss / refine separately.